forge/test/mlir/test_features.py

```diff
@@ -10,7 +10,7 @@
 from torch import nn
 
 import forge
-from forge.op.eval.common import compare_with_golden_pcc
+from forge.op.eval.common import compare_with_golden_pcc, compare_with_golden
```

Comment:
Unrelated to current PR - we should establish a proper way to use compare_with_golden - seems a bit clunky to import both here.